Contributors.md

```diff
@@ -0,0 +1,42 @@
+Since its first C++ prototypes many people and groups have contributed to GLS, from bug reports to major feature additions.
+If your name is missing from the list please send an email to jogol@microsoft.com.
+
+Names are sorted within groups by last name.
+Groups are in order of commit magnitude.
+
+
+#### [Josh Goldberg](http://github.com/JoshuaKGoldberg)
+* Project lead
+
+
+### [RCOS](https://rcos.io/projects/HighSchoolHacking/GLS/profile) RPI Students
+
+* [Carter Gleason](https://github.com/GleasonC4)
+    * Fall 2016:
+        * Null checking [design](https://github.com/HighSchoolHacking/GLS/pull/126]) and [implementation](https://github.com/HighSchoolHacking/GLS/pull/135)
+        * For each [design](https://github.com/HighSchoolHacking/GLS/pull/138) and [implementation](https://github.com/HighSchoolHacking/GLS/pull/143)
+        * [Exceptions](https://github.com/HighSchoolHacking/GLS/pulls?utf8=%E2%9C%93&q=is%3Apr%20exceptions)
+        * Various code cleanups (dictionaries, comment blocks, files, main, while, print)
+        * JavaScript language support with Benjamin Wolf
+* [Benjamin Wolf](https://github.com/Ben-Wolf)
+    * Fall 2016:
+        * [List pops design and implementation](https://github.com/HighSchoolHacking/GLS/pull/131)
```

Comment:
List Pop is still a bit odd, might I suggest. 
`List pops [design](https://github.com/HighSchoolHacking/GLS/pull/128) and [implementation](https://github.com/HighSchoolHacking/GLS/pull/131)`